lib/rules/no-async-in-computed-properties.js

```diff
@@ -77,13 +77,16 @@ module.exports = {
   },
   /** @param {RuleContext} context */
   create(context) {
+    /** @type {Map<ObjectExpression, ComponentComputedProperty[]>} */
+    const computedPropertiesMap = new Map()
+    /** @type {Set<FunctionExpression | ArrowFunctionExpression>} */
+    const computedFunctionNodes = new Set()
```

Comment:
You can probably do the same with an array. Since the array is simpler, can you change it to an array?